pkg/datapath/loader/loader.go

```diff
@@ -132,6 +137,63 @@ func nullifyStringSubstitutions(strings map[string]string) map[string]string {
 	return nullStrings
 }
 
+// patchHostNetdevHeaderFile creates a header file corresponding to each new
+// device to which bpf_host is attached, including the second host device and
+// native devices.
+func patchHostNetdevHeaderFile(stateDir, ifName string, mac mac.MAC, ifIndex uint32,
+	secctxFromIpcache uint32, nodePort bool) (netdevHeaderFile string, err error) {
```

Comment:
This func seems to use a mixture of named returns and explicit returns. IMO for `(... , error)` funcs explicit returns should be preferred, see also https://github.com/golang/go/wiki/CodeReviewComments#named-result-parameters

i.e.

```suggestion
	secctxFromIpcache uint32, nodePort bool) (string, error) {
```

and then change the naked returns below to `return nil, err` which will btw also make sure that the correct, unshadowed `err` will be returned.

Reply:
Sorry, I think my claim about shadowing `err` here is incorrect. But I still think consistently using explicity returns makes the func more readable.